frontend/about.html

```diff
@@ -109,74 +109,135 @@ <h2 class="page_title ">ABOUT</h2>
+            <br>
+            <p>The Open Data for Resilience Index is managed by the <a href="https://opendri.org" target="_blank">Open Data for Resilience Initiative</a> (OpenDRI). Together with other tools such as <a href="http://thinkhazard.org" target="_blank">ThinkHazard!</a>, <a href="http://geonode.org/" target="_blank">Geonode</a>, and <a href="">Inasafe</a>, it aims at improving risk information through better access to data.</p>
 
-        <p>Information available on the website is collected and updated on a voluntary basis primarily by people from the <a href="https://understandrisk.org" target="_blank">Understanding Risk</a> (UR) community and then reviewed by a team of Disaster Risk Management and Open Data specialists.</p>
+            <p>Information available on the website is collected and updated on a voluntary basis primarily by people from the <a href="https://understandrisk.org" target="_blank">Understanding Risk</a> (UR) community and then reviewed by a team of Disaster Risk Management and Open Data specialists.</p>
 
-        <h2>OpenDRI</h2>
+            <h2 id="opendri">OpenDRI</h2>
 
-        <p>In 2011, the <a href="https://www.gfdrr.org" target="_blank">Global Facility for Disaster Reduction and Recovery</a> (GFDRR) launched the Open Data for Resilience Initiative to apply the concepts of the global open data movement to the challenges of reducing vulnerability to natural hazards and the impacts of climate change. OpenDRI supports <a href="http://www.worldbank.org/en/topic/disasterriskmanagement" target="http://www.worldbank.org/en/topic/disasterriskmanagement">World Bank Regional Disaster Risk Management Teams</a> to build capacity and long-term ownership of open data projects with client countries that are tailored to meet specific needs and goals of stakeholders.</p>
+            <p>In 2011, the <a href="https://www.gfdrr.org" target="_blank">Global Facility for Disaster Reduction and Recovery</a> (GFDRR) launched the Open Data for Resilience Initiative to apply the concepts of the global open data movement to the challenges of reducing vulnerability to natural hazards and the impacts of climate change. OpenDRI supports <a href="http://www.worldbank.org/en/topic/disasterriskmanagement" target="http://www.worldbank.org/en/topic/disasterriskmanagement">World Bank Regional Disaster Risk Management Teams</a> to build capacity and long-term ownership of open data projects with client countries that are tailored to meet specific needs and goals of stakeholders.</p>
+
+            <!-- Back to the top -->
+            <div class="text-center">
+                <a href="#startReading"><span class="small"><span class="small text-muted"><i class="fa fa-arrow-up" aria-hidden="true"></i> top</span></span></a>
```

Comment:
The [HTML5 specification](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-href) states that using `#top` or `#` actually directs the user to the top of the document.

The `startReading` id can be removed and the `href` can be updated to whichever of these values.